backend/danswer/indexing/models.py

```diff
@@ -108,7 +108,9 @@ def from_model(
         embedding_model: "EmbeddingModel",
     ) -> "EmbeddingModelDetail":
         return cls(
-            model_name=embedding_model.model_name,
+            # When constructing EmbeddingModel Detail for user-facing flows, strip the
+            # unneeded additional data after the `_`s
+            model_name=embedding_model.model_name.split("__")[0],
```

Comment:
I slightly prefer using something that is basically impossible to "get unlucky" with. Like suffixing with "__danswer_alt_index" instead of __alt and checking here just for __

Reply:
__danswer_alt_index can be a globally set constant string as well just for consistency